exchanges/sharedtestvalues/sharedtestvalues.go

```diff
@@ -47,3 +55,48 @@ func NewTestWebsocket() *stream.Websocket {
 		Match:             stream.NewMatch(),
 	}
 }
+
+// SkipUnsetCredentials is a test helper function checking if the authenticated function
+// can perform the required test.
+func SkipUnsetCredentials(t *testing.T, exch exchange.IBotExchange, canManipulateOrders ...bool) {
```

Comment:
`SkipTestIfCredentialsUnset`
Otherwise it implies you're skipping a test and unsetting a user's credentials